Added it all back (#336)
sunpy/sunpy.org: project/roles.rst

**Estimated Commitment Level**: Around 1 hour per week, but comes in bursts as PRs are opened.

.. _role_maintainer:

Maintainer
----------

Responsibilities of a maintainer (subpackage, or library etc) include:

* Evaluating and soliciting new pull requests which are consistent with their scope and contribution standards.
* Providing material support for open pull requests to enable merging.

> * Maintaining and developing the roadmap consistent with the roadmap of the overall core package.

Reviewer: ```suggestion
* Maintaining and developing the roadmap consistent with the roadmap of the project.
```

* Mentoring the next generation of maintainers and developers.
* Responding to relevant queries on all official SunPy communication channels.
* Handling GitHub issues associated with their scope.

> * Overseeing and improving documentation content.

Reviewer: ```suggestion
* Overseeing and improving related documentation content.
```

> * Soliciting and implementing improvements and new additions to the documentation content.

Reviewer: ```suggestion
* Soliciting and implementing improvements and new additions to related documentation content.
```
